src/logic/Utils.ts

```diff
@@ -242,8 +242,8 @@ export const injectColourScheme = (scheme: ColourScheme | null) => {
     let injectStr = ":root {";
     const themeIniKeys = Object.keys(scheme);
     themeIniKeys.forEach((key) => {
-      injectStr += `--spice-${key}: #${scheme[key]} !important;`;
-      injectStr += `--spice-rgb-${key}: ${hexToRGB(scheme[key])} !important;`;
```

Comment:
#287

Reply:
Actually we dont need that `!important` because this PR actually removes the `color.css` like `user.css` used to get removed when installing a theme from marketplace.
![image](https://user-images.githubusercontent.com/75513645/215842662-91a4467c-69a3-4e78-94df-9e6c1fe3cbb0.png)

Marketplace devs just forgot to remove `color.css` while installing a theme from marketplace as they actually wrote code to re-inject `color.css` again when a theme is uninstalled from marketplace.
![image](https://user-images.githubusercontent.com/75513645/215843035-1d3dca69-5763-433e-b9e8-be01cf931125.png)

#287 @kyrie25 just fixed this problem by adding a `!important` instead of removing the old `color.css`

Reply:
Ok i see that you removed those lines and devs didnt forget that. I dont know why you removed it. But removing `color.css` should remove all the `--spice-something` variable

### U can see there is no `:root` that contains `--spice-something` variables
https://user-images.githubusercontent.com/75513645/215853558-d9371c90-f0e5-41be-83d2-42fae83ffb32.mp4

Other that `color.css` where do you find `--spice-something` variables kyrie?